rush.json

```
@@ -203,6 +203,12 @@
       "projectFolder": "packages/ref-doc",
       "reviewCategory": "production",
       "shouldPublish": false
+    },
+    {
+      "packageName": "@typespec/json-schema",
+      "projectFolder": "packages/json-schema",
+      "reviewCategory": "production",
+      "shouldPublish": true
```

```suggestion
      "versionPolicyName": "typespec"
```

think need that if we want to pin it to the rest